src/styles/variables.scss

```diff
@@ -0,0 +1,20 @@
+$white: #ffffff;
+$black: #333031;
+$ui-background-01: #eeeeee;
+$ui-background-02: #eeeeee;
```

Comment:
This confused me when I added the colors to generogrowth. This one used `rgb(238 238 238 / 50%)` while the styleguide defined it as `rgb(238 238 238 / 75%)` but then when I started using the color it seemed like the designer had nested it on top of $ui-background-01, so I wonder what it's actually supposed to be :D

Reply:
I'll ask.